content/en/blog/_posts/2022-12-15-dynamic-resource-allocation-alpha/index.md

```diff
@@ -217,26 +217,26 @@ business logic:
 
 - [k8s.io/dynamic-resource-allocation/controller](https://github.com/kubernetes/dynamic-resource-allocation/tree/release-1.26/controller)
 
-Likewise, boilerplate code can be used to register the node-local plugin with
-the kubelet, as well as start a gRPC server to implement the kubelet plugin
-API. For drivers written in Go, the following package is recommended:
+  Likewise, boilerplate code can be used to register the node-local plugin with
+  the kubelet, as well as start a gRPC server to implement the kubelet plugin
+  API. For drivers written in Go, the following package is recommended:
```

Comment:
@windsonsea, why this change?

Reply:
Seems this is an unordered list with two items. The `Likewise` para is to describe the first item. Is my understanding wrong?

<img width="763" alt="image" src="https://user-images.githubusercontent.com/79828097/211953587-384ed800-3586-4988-b065-3c537a13ae74.png">

Reply:
In my opinion, the paragraph doesn't need an extra indentation. 

Before the first bullet point, you have: "As such, Kubernetes **_provides the following package_**, including APIs for invoking this boilerplate code as well as a `Driver` interface that you can implement to provide their custom business logic: -> first bullet point

Before the second bullet point, you have: "Likewise, boilerplate code can be used to register the node-local plugin with the kubelet, as well as start a gRPC server to implement the kubelet plugin API. For drivers written in Go, **_the following package is recommended_**: -> second bullet point

Both bullet points show a link to download some source code related to the preceding paragraph.

@windsonsea @sftim

Reply:
Well noted. This section is restored.